src/protocol/thread/pause.ts

```diff
@@ -451,9 +451,18 @@ export class Pause {
       return this.loadMouseTargetsWaiter.promise;
     }
     this.loadMouseTargetsWaiter = defer();
-    const { elements } = await this.sendMessage(client.DOM.getAllBoundingClientRects, {});
-    this.mouseTargets = elements;
-    this.loadMouseTargetsWaiter.resolve();
+    let rv;
+    try {
+      rv = await this.sendMessage(client.DOM.getAllBoundingClientRects, {});
+      this.mouseTargets = rv.elements;
+    } catch (e) {
+      console.error("DOM.getAllBoundingClientRects failed", e);
+    } finally {
+      this.mouseTargets = [];
+      this.loadMouseTargetsWaiter.resolve();
+    }
+
+    return rv;
```

Comment:
rv here is basically some random object.  This function should return a boolean indicating if it failed or not.

Reply:
It would be nice if this returned !!rv so the return value is a boolean.